docs-assets/app/app/Livewire/Forms/LayoutDemo.php

```diff
@@ -300,6 +300,33 @@ public function form(Form $form): Form
                             ])
                             ->columns(3),
                     ]),
+                Group::make()
+                    ->id('sectionHeaderActions')
+                    ->extraAttributes([
+                        'class' => 'p-16 max-w-2xl',
+                    ])
+                    ->schema([
+                        Section::make('Rate limiting')
+                            ->description('Prevent abuse by limiting the number of requests per period')
+                            ->headerActions([
+                                Action::make('test'),
```

Comment:
Shouldn't this also be "edit", @danharrin?
```suggestion
                                Action::make('edit'),
```

Reply:
Sorry, no, it makes sense as is.